### ISSUE TITLE
Inconsistent Multiple Commands: Non Set commands must return different error message

### ISSUE THREAD

**Issue by apoorvyadav1111** [DiceDB/dice#411]
Non Set related commands such as `GET, GETDEL, INCR...` is not consistent with Redis implementation. Here are the steps to reproduce the issue

```
sadd foo bar
```

Here's the output I observed in Redis v7.2.5, if we run 
```
get foo
``` 
or 
```
getdel foo
```
in Redis v7.2.5
```
(error) WRONGTYPE Operation against a key holding the wrong kind of value
```

and here's the output I observed in DiceDB's latest commit of the `master` branch

```
(error) ERR expected string but got another type
```

Make the implementation consistent with the Redis implementation.
Make sure you are using Redis version 7.2.5 as a reference for the
command implementation and to setup Redis

I am working on finding more about the inconsistent commands that need to handle sets explicitly. I will update this issue and related PR with the commands affected.

- [from source code](https://gist.github.com/arpitbbhayani/94aedf279349303ed7394197976b6843), or
- [use Docker](https://hub.docker.com/_/redis).

**Comment by apoorvyadav1111** [comment]
Hi @JyotinderSingh , Please assign this to me.

**Comment by JyotinderSingh** [comment]
> Hi @JyotinderSingh , Please assign this to me.

Done

**Comment by lucifercr07** [comment]
@apoorvyadav1111 looks like this issue has been fixed as part of #413. Shall we close this?

**Comment by apoorvyadav1111** [comment]
Yes, we can close this issue.
